pkg/util/util.go

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package util
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/k8sclient"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"math/rand"
+	"os"
+)
+
+// GetEnvValue looks for env variables in Operator pod to configure Code Ready deployments
+// with things like db users, passwords and deployment options in general. Envs are set in
+// a ConfigMap at deploy/config.yaml. Find more details on deployment options in README.md
+func GetEnv(key, defaultValue string) string {
+	value := os.Getenv(key)
+	if len(value) == 0 {
```

Comment:
What if someone decided to set an empty value? Would this function behave correctly?

Reply:
Hm.. good question. Let me experiment a bit and I'll write up a respose